doc/Developing.md

```diff
@@ -62,6 +73,63 @@ It can also be started with:
 
     $ npm start
 
+## Synchronizing changes between theia and the examples
+
+The examples provided with theia are structured such that they have a file
+dependency on the theia root directory, the component in this repository is
+called theia-core.
+
+This means that the examples have their own version of theia-core in their
+node_modules directory.
+
+So each time you make a change in the theia-core you need to synchronize
+the theia-core in the example's node_modules directory.
+
+The current way to do so is to run this command in the example directory:
+
+	$ rm -rf node_modules/theia && npm install
```

Comment:
Note I just retried and there's nothing to be done :( the file-updater sees the change but no hot-reload, reloading the page doesn't help either... but we can discuss this as part of #125